### ISSUE TITLE
Opening Applications view causes artifacts on secondary monitor

### ISSUE THREAD

**Issue by leviport** [pop-os/gnome-shell-extension-pop-cosmic#88]
To reproduce:
- Have multiple windows open on secondary monitor
- Open Applications with button on top bar or Super+A

If I first open Workspaces, then open Applications, all of the windows on my second monitor disappear, showing my wallpaper. I think the ideal default behavior is to not change any of the windows on the second monitor. My primary display has the gray overlay with the applications grid, so why should the secondary display change at all?

![PXL_20210623_223824300](https://user-images.githubusercontent.com/13512097/123177095-e0897680-d441-11eb-9531-114631dc1c89.jpg)

(screenshots did not capture the artifacts, so I had to take a picture of my display)

**Comment by Avendattor** [comment]
I have same problem.

Upd:
In my opinion, second screen behavior in App view should be same as Workspaces view.

**Comment by Kartonrealista** [comment]
#86 probably the same issue, it's being worked on

**Comment by WatchMkr** [comment]
@ids1024 the ideal behavior is that there are no animations or changes to windows on secondary monitors.

**Comment by jackpot51** [comment]
Does this happen with COSMIC extension disabled?

We can either fix this specific issue, or work on the new behavior of not doing the window spread on Applications.

**Comment by jacobgkau** [comment]
This specific issue (and #86) is fixed with @ids1024's commit in the `app-grid-animation` branch. That commit introduces another regression, but reverting [part of it](https://github.com/pop-os/cosmic/commit/ca6510a93140a73b2b13398c32a32e35f7c89797#r52610467) fixes the issues without the regression. Just wanted to make that clear since the commit's hidden without a PR.

**Comment by Kartonrealista** [comment]
> Does this happen with COSMIC extension disabled?

This is actually pretty funny. I disabled COSMIC and this happened:
![isuue](https://user-images.githubusercontent.com/63356574/123296231-c40b3e00-d516-11eb-8064-9e95b0a9f80d.gif)
First I opened activities with SUPER and then apps with SUPER + A. It doesn't show any apps, just dissapears the workspace picker. It seems like that's a separate issue alltogether but just in case it has something to do with this I'll mention it. It doesn't happen when I turn off the Ubuntu Dock extension.
